### DISCUSSION TITLE
[Popover][Tooltip] custom arrow

### DISCUSSION THREAD

**Discussion by SombreroElGringo** [radix-ui/primitives#1828] [Help] [answered]
Hello,

I am trying to use a custom arrow svg for my tooltips and popovers, but cannot found a way to implement it. Do you have some suggestion how we can achieve this?

I found this example done on an older version of the popover component, but when I tried with the latest version it seems it was not working anymore 

https://github.com/radix-ui/primitives/discussions/868#discussioncomment-1342234

**Comment by benoitgrelard** [comment, marked as answer]
Yeah you can use `asChild` on the arrow part.
So:
```tsx
<Popover.Arrow asChild>
  <MySvgArrow />
</Popover.Arrow>
```

> Note: Your SVG arrow needs to be pointing down by default
